test/test_files/randomprime_expected_data.json

```diff
@@ -11,7 +11,8 @@
         "qolPickupScans": false,
         "quickplay": false,
         "quiet": false,
-        "trilogyDiscPath": null
+        "trilogyDiscPath": null,
+        "suitColors": {}
```

Comment:
I think it's ideal to alter the cosmetic patches used in the test case so the custom colors are used.